antora-ui-camel/src/css/vars.css

```diff
@@ -131,11 +131,14 @@
   --footer-height: 23rem; /* empirically corresponding current design of the foooter */
   /* dimensions */
   --navbar-height: calc(73 / var(--rem-base) * 1rem);
+  --toolbar-nav-height: calc(139.5 / var(--rem-base) * 1rem);
+  --panel-height: calc(54 / var(--rem-base) * 1rem);
   --toolbar-height: calc(45 / var(--rem-base) * 1rem);
   --drawer-height: var(--toolbar-height);
   --body-top: calc(var(--navbar-height) + 1rem);
   --body-min-height: calc(100vh - var(--navbar-height));
   --nav-height: calc(var(--body-min-height) - var(--toolbar-height));
+  --nav-menu-panel-height: calc(100vh - var(--panel-height));
   --nav-height--desktop: var(--body-min-height);
   --nav-panel-height: calc(var(--nav-height) - var(--drawer-height));
```

Comment:
This value needs to be adjusted, the height of the search bar needs to be subtracted from the height to fit the explore drawer